client/src/components/LinkButton.tsx

```diff
@@ -65,14 +69,28 @@ export default function LinkButton(props: Props) {
     if (error != null && error.error_code === 'INVALID_LINK_TOKEN') {
       await generateLinkToken(props.userId, props.itemId);
     }
-
     // to handle other error codes, see https://plaid.com/docs/errors/
   };
 
+  const onEvent = async (
+    eventName: PlaidLinkStableEvent | string,
+    metadata: PlaidLinkOnEventMetadata
+  ) => {
+    // handle errors
+    if (
+      eventName === 'ERROR' &&
+      metadata.error_code != null &&
+      metadata.institution_name != null
```

Comment:
Is there a world where `error_code` is NOT `null` but `institution_name` is? I assume we'd still want to throw an error in these instances.

Reply:
done!